libs/voicefocus/voicefocus.js

```diff
@@ -23,12 +26,12 @@ const DEFAULT_AGC_DISABLED_SETTING = {
 };
 const DEFAULT_AGC_SETTING = DEFAULT_AGC_DISABLED_SETTING;
 const DEFAULT_ASSET_GROUP = 'stable-v1';
-const DEFAULT_CDN = 'https://static.sdkassets.chime.aws/';
+const DEFAULT_CDN = 'https://static.sdkassets.chime.aws';
 const DEFAULT_PATHS = {
-    processors: `${DEFAULT_CDN}processors/`,
-    workers: `${DEFAULT_CDN}workers/`,
-    wasm: `${DEFAULT_CDN}wasm/`,
-    models: `${DEFAULT_CDN}wasm/`,
+    processors: `${DEFAULT_CDN}/processors/`,
```

Comment:
just curious, what is the benefit of this change?

Reply:
No benefit except I think it's cleaner / easier to read.